app/models/manageiq/providers/redhat/infra_manager/refresh_parser.rb

```diff
@@ -658,4 +659,10 @@ def self.datacenter_inv_to_hashes(inv, cluster_uids, vm_uids, storage_uids, host
 
     result
   end
+
+  private_class_method
```

Comment:
Although this method is only used in this class, which would make it a private method,
most of the other methods in that class are also only used in this class.
And this pattern is the same for the other refresh parsers as well.

So, for the sake of consistency I would follow the pattern here.

But fee free to come up with a follow up PR, that cleans this up with the pattern described in this [post](http://blog.codeclimate.com/blog/2012/11/14/why-ruby-class-methods-resist-refactoring/)